content/post/release-of-1.8.0.md

```diff
@@ -0,0 +1,435 @@
+---
+date: "2019-04-21T10:40:00+01:00" 
+author: "zeripath"
+title: "Gitea 1.8.0 is released"
+tags: ["release"]
+draft: false
+---
+
+The time has come for another major release! We are proud to present Gitea
+1.8.0 to the world. **This release contains two new security fixes which cannot be backported to the 1.7.0 branch, so it is recommended to update to this version.**
+In this release, we merged
+[245 pull requests](https://github.com/go-gitea/gitea/pulls?utf8=%E2%9C%93&q=is:pr+milestone:1.8.0+is:merged)
+-- This is more than 1.7.0 (157) and 1.6.0 (178).
+
+You can download one of our pre-built binaries from our
+[downloads page](https://dl.gitea.io/gitea/1.8.0/) - make sure to select the
+correct platform! For further details on how to install, follow our
+[installation guide](https://docs.gitea.io/en-us/install-from-binary/). Unfortunately
+there is a known issue with our prebuilt images on armv7, see [#6700](https://github.com/go-gitea/gitea/issues/6700).
+
+We'd like to thank all of our backers on
+[Open Collective](https://opencollective.com/gitea), who are helping us deliver
+a better piece of software.
+
+With that out of the way, here's what's new in Gitea version 1.8.0:
+
+<!--more-->
+
+## OAuth2 Provider [#5378](https://github.com/go-gitea/gitea/pull/5378)
+
+Gitea is now an [OAuth2 Provider](https://docs.gitea.io/en-us/oauth2-provider/). This fixes and closes issue [#27](https://github.com/go-gitea/gitea/issues/27)!
+
+![Authorize TestApp Access](https://docs.gitea.io/authorize.png)
+
+_Thanks to [**@jonasfranz**](https://github.com/jonasfranz)_
+
+## Repositories can now be archived [#5009](https://github.com/go-gitea/gitea/pull/5009)
+
+Repositories in Gitea can now be archived. This prevents the creation of new issues,
+pull-requests or pushing to the git repository.
+
+![Archived Repository](/demos/5009/1.png)
+
+![Archived Repository List](/demos/5009/2.png)
+
+_Thanks to [**@kolaente**](https://github.com/kolaente)_
+
+## Private and Internal Organizations [#1763](https://github.com/go-gitea/gitea/pull/1763)
+
+Organizations can now be set to be Public, Internal or Private. Repositories created in
+internal organizations will only be visible to logged-in users. Private organizations will
+only be visible to those who are members.
+
+![Archived Repository List](/demos/1763/1.png)
+
+_Thanks to [**@techknowlogick**](https://github.com/techknowlogick) &amp; [**@DblK**](https://github.com/dblk)_
+
+## Implement "conversation lock" for issue comments [#5073](https://github.com/go-gitea/gitea/pull/5073)
+
+Conversations can now be locked in similar way to Github.
+
+![Locked Comments](/demos/5073/1.png)
```

Comment:
This screenshot could at least be resized if not swapped...

Reply:
done